packages/gluestick/src/config/webpack/ChunksPlugin.js

```diff
@@ -41,18 +32,18 @@ const getChunksInfoBody = (json: Object, publicPath: string): ChunksInfo => {
 
     return chunk
       .filter(name => path.extname(name) === `.${extension}`)
-      .map(name => `${publicPath}${name}`);
+      .map(name => ({ url: `${publicPath}${name}`, name }));
   };
 
   Object.keys(assetsByChunk).forEach((name: string) => {
     // The second asset is usually a source map
-    const jsAsset: string = getAssets(name, 'js')[0];
+    const jsAsset = getAssets(name, 'js')[0];
```

Comment:
why no more flow typing?

Reply:
commented a bit late: most of the Flow types in this file are redundant and already inferred. I improved typing overall by getting rid of some `Object` ones.